app-creator/src/widgets/ui-slider/ui-slider.ts

```diff
@@ -29,24 +30,53 @@ export class Slider extends LitElement {
    */
   @property({ type: Number }) max = 100;
 
+  /**
+   * Sets minimum slider value.
+   */
+  @property({ type: Number }) min = 0;
+
   /**
    * Displays input field next to slider for editing.
    */
-  @property({ type: Boolean }) editable = false;
+  @property({ type: Boolean }) editable = true;
 
   render() {
-    const { value, max, editable, styles } = this;
+    const { value, max, min, editable, disabled, styles } = this;
     return html`
       <paper-slider
         style=${styleMap(styles)}
         value=${value}
         max=${max}
+        min=${min}
+        ?disabled=${disabled}
         ?editable=${editable}
       >
       </paper-slider>
     `;
   }
 
+  setAttribute(key: string, value: string) {
+    switch (key) {
+      case 'value':
+        this.value = parseInt(value);
```

Comment:
You may want to check for NaNs and provide default values, and for below.

Reply:
Since it's a number input, the values cannot be invalid. I've added a check to make sure we don't add negative numbers either.